docs/guides/rdl-metadata.md

```diff
@@ -0,0 +1,11 @@
+# RDL metadata
```

Comment:
As this page is unfinished, I would remove it for now and put the draft content in a GitHub issue.

Reply:
removed reference from docs/guides/index.md, created issue #99